lab-yunjoo/app/component/note-li/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+require('./note-li.scss');
+const angular = require('angular');
+angular.module('demoApp').directive('appNoteLi', function(){
+  return {
+    restrict: 'E',
+    replace: true,
+    template: require('./note-li.html'),
+    controller: ['$log', 'noteService', NoteLiController],
+    controllerAs: 'noteLiCtrl',
+    bindToController: true,
+    scope: {
+      note: '=',
+      deleteNote: '&'
+    }
+  };
+});
+
+function NoteLiController($log,noteService){
```

Comment:
It's probably best not to name things after the html tag they are attached to, but after what they are designed to do.